docs/hsa/hsa.rst

```diff
@@ -0,0 +1,187 @@
+.. doctest-skip-all
```

Comment:
You can remove this line, so this docs will be tested. For all examples that are accessing the server, please add the `.. doctest-remote-data::` directive.

We're in the middle to doing this for all other modules retrospect, you can find plenty of examples for in the `docs` directory of how to use the directive and the different ignore and skip options.

Reply:
Done. I followed some of the other docs. Let me know if there is something I need to change.